src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.css

```diff
@@ -2,16 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-.breakpoint-panel-wrapper {
-  min-width: 300px;
-}
-
 .breakpoint-panel {
-  margin: 4px 0;
+  margin: 0.25rem 0 0.5rem -1px;
```

Comment:
The negative pixel here pulls the panel back slightly so it doesn't create a 1px space between the gutter and panel.